### PR TITLE
Enable non-googlers to run anago

### PR DESCRIPTION
**Pull request by jbeda**
This is part of #268.

### CODE REVIEW

File: lib/releaselib.sh [hunk 1]
```diff
@@ -111,7 +111,8 @@ release::set_build_version () {
   # kubernetes-e2e-gke-subnet - Uses a branch version?
   # kubernetes-e2e-gke-test - Uses a branch version?
   local -a gke_jobs=(
-                     "ci-kubernetes-e2e-gke-serial$branch_suffix"
+                     # This job has been failing for quite a while.
+                     # "ci-kubernetes-e2e-gke-serial$branch_suffix"
```
Comment (ixdy) [inline lib/releaselib.sh]: This one has actually started passing again, finally.

**Comment by jbeda** [comment]
@ixdy -- picking this up again. Please take another look!  Thanks.

**Comment by jbeda** [comment]
@ixdy -- is there a robot or something to merge this?  Or should I self merge?  What is the etiquette here?

**Comment by ixdy** [comment]
this repo is self-merge.

**Pull request merged**